llvm/include/llvm/Object/Binary.h

```diff
@@ -202,6 +202,7 @@ template <typename T> class OwningBinary {
   OwningBinary(std::unique_ptr<T> Bin, std::unique_ptr<MemoryBuffer> Buf);
   OwningBinary(OwningBinary<T>&& Other);
   OwningBinary<T> &operator=(OwningBinary<T> &&Other);
+  void operator()(OwningBinary<T> &&Other);
```

Comment:
why is this a function call operator? How is this different from the move operator above?